crates/flox-rust-sdk/src/models/environment/managed_environment.rs

```diff
@@ -587,7 +587,7 @@ impl ManagedEnvironment {
         temp_floxmeta.git.push("origin").unwrap();
 
         fs::write(
-            temp_floxmeta_path.join("env.json"),
+            path_environment.path.join("env.json"),
```

Comment:
suggestion nonblocking: should we also be removing `path_environment.path.join("env")`?